.buildkite/browser-pipeline.yml

```diff
@@ -16,6 +16,111 @@ steps:
       - docker-compose#v3.9.0:
           push:
             - browser-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-browser-${BRANCH_NAME}
+#
+  - label: ':chrome: v30 Browser tests'
```

Comment:
I don't think we want to add all these version of Chrome.  Adding them to Maze Runner was just in case we found a problem with v30, but if 30 passes then I would remove 32 to 42.